Automattic/jetpack
# Jetpack Boost: Handle zero dimensions in getOversizedRatio

Fixes #32037

## Proposed changes:
- Added comments to clarify why we return 1 when either dimension of the image is 0 in getOversizedRatio function
- Clarified why we only initialize the image guide after the window has loaded in a comment

### Other information:



## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
N/A

## Testing instructions:
- Load an image with 0 width or height and ensure the oversized ratio returned is 1
- Check the guide doesn't initialize in an iframe
- Load a page where the image guide is expected, ensure it doesn't initialize until the window is fully loaded
